File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/RibbonBar/Appearance/AddingPagePage.vb
Namespace CompanionKit.Controls.RibbonBar.Appearance

	Public Class AddingPagePage
		Inherits UserControl

		Public Sub New()
			Me.InitializeComponent()
			Me.demoRibbonBar.SelectedIndex = 0
			Me.SelectedIndexChanged(Me.demoRibbonBar, New EventArgs)
		End Sub

		Private Sub SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs)
			If (Me.demoRibbonBar.SelectedIndex > -1) Then
				Dim page As RibbonBarPage = Me.demoRibbonBar.Pages.Item(Me.demoRibbonBar.SelectedIndex)
				If (Not page Is Nothing) Then
					Me.lblStatus.Text = String.Format("The active page: {0}", page.Text)
				End If
			End If
		End Sub

	End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/RibbonBar/Appearance/AddingPagePage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.RibbonBar.Appearance
{
    public partial class AddingPagePage : UserControl
    {
        public AddingPagePage()
        {
            InitializeComponent();

			demoRibbonBar.SelectedIndex = 0;
			SelectedIndexChanged(demoRibbonBar, new EventArgs());
        }

		/// <summary>
		/// Handle RibbonBarPage activation
		/// </summary>
		private void SelectedIndexChanged(object sender, EventArgs e)
		{
			if (demoRibbonBar.SelectedIndex > -1)
			{
				RibbonBarPage objPage = demoRibbonBar.Pages[demoRibbonBar.SelectedIndex];
				if (objPage != null)
				{
					lblStatus.Text = string.Format("The active page: {0}", objPage.Text);
				}
			}
		}
    }
}